Sources/Runner/Commands/RunDangerJS.swift

```diff
@@ -18,8 +18,17 @@ func runDangerJSCommandToRunDangerSwift(_ command: DangerCommand, logger: Logger
     proc.environment = ProcessInfo.processInfo.environment
     proc.launchPath = dangerJS
 
-    let unusedArgs = CommandLine.arguments.filter { arg in
-        !arg.contains("danger-swift") && arg != command.rawValue
+    let dangerOptionsIndexes = DangerSwiftOptions.allCases.compactMap { ($0, CommandLine.arguments.firstIndex(of: $0.rawValue)) }
```

Comment:
<!--
  0 failure: 
  1 warning:  Line should be 12...
  
  
  DangerID: danger-id-default;
  File: Sources/Runner/Commands/RunDangerJS.swift;
  Line: 21;
-->

- :warning: Line should be 120 characters or less: currently 129 characters (`line_length`)